assets/data-port/import/data/reducer.js

```diff
@@ -168,12 +179,15 @@ export default ( state = DEFAULT_STATE, action ) => {
 		case SET_STEP_DATA:
```

Comment:
Without the resolver, I think this is only used in `fetchImportLog`. I wonder if we should just rename the action, constant, reducer use to `SUCCCESS_FETCH_IMPORT_LOG`?

Reply:
Replaced in 72528b2